index.html

```diff
@@ -1041,194 +1041,38 @@
-probably unique to the person.
-
-The attributes can be exposed as information about the [=person=]'s device
-that on their own are benign (as opposed to [[[#hl-sensitive-information]]]).
-Taken in the aggregate these attributes could uniquely identify the device, or
-contribute to possible [=cross-context recognition=].
-For example:
+People may wish to present an ephemeral or anonymous identity. This is
+a set of characteristics that is too small or unstable to be useful
+for following them through time.
 
-* language and time zone;
-* window size;
-* system preferences (such as dark mode, serif font, etc.).
+A person's [=identities=] may often be distinct from whatever legal identity
+or identities they may hold.
 
-Preventing [=fingerprinting=] can be particularly challenging in cases that
-only affect a small group of people who use the web. For example, people who
-configure their systems in unique ways, such as by using a browser with a very
-small number of users.
-See [[fingerprinting-guidance]] for how to mitigate threats that result from
-[=fingerprinting=].
+In some circumstances, the best way for a [=user agent=] to uphold this
+principle is to prevent [=recognition=] (i.e. so that one [=site=] can't
+learn anything about its [=user=]'s behavior on *another* site).
 
-Supercookies occur when a user agent stores data for a site but makes that data more
-difficult to clear than other cookies or storage, typically because of a bug, of features
-relating to cache storage and network state (e.g. ETag, HSTS), or
-because the browser restores the browser vendor's cookies when local state is cleared.
-<a data-cite="fingerprinting-guidance#clearing-all-local-state">Fingerprinting
-Guidance § Clearing all local state</a> discusses how specifications can help
-user agents avoid this mistake.
+In other circumstances, the best way for a [=user agent=] to uphold this
+principle is to *support* [=recognition=] (i.e. to help its [=user=] prove
+to one [=site=] that they have a particular identity on another [=site=]).
 
-<dfn>Header enrichment</dfn> happens when a network operator adds HTTP request headers
-to identify their customers to sites that they visit. It is unfortunately
-difficult for a [=user agent=] to mitigate against [=header enrichment=].
+Similarly, a [=user agent=] can help its [=user=] by preventing or supporting
+[=recognition=] across *repeat* visits to the *same* [=site=].
 
-<dfn>Cross-device communication</dfn> is communication
-between code on one device and code running on another device. For example, sounds or
-light emitted from one device could be detected by a microphone or light sensor on
-another device [[?SILVERPUSH]]. [=Cross-device communication=] enables cross-device
-tracking, a form of [=cross-context recognition=], but it can also be used for other
-inappropriate information flows.
+[=User agents=] should do their best to distinguish [=contexts=] within a site,
+and adjust their [=partitions=] when necessary.
```

Comment:
On the fence about this. I haven't been assuming people will read the document from top to bottom in order (who does that??) and partitions is linked to a definition for anyone who hasn't read that bit yet or doesn't know the concept. I am hoping we can avoid re-explaining everything everywhere, as if each section needs to completely stand on its own.

That said, happy for the group to resolve this suggestion without me today!

Reply:
```suggestion
[=User agents=] should do their best to distinguish [=contexts=] within a site
and adjust their [=partitions=] to prevent or support [=recognition=] across those intra-site [=contexts=]
according to their [=users=]' wishes.
```